src/styles/ProjectEditDialog.scss

```diff
@@ -0,0 +1,13 @@
+.ProjectEditDialog {
+    .MuiDialog-paper {
+        margin: 16px
+    }
+}
+
+@media screen and (min-width: $sm) {
+    .ProjectEditDialog {
+        .MuiDialog-paper {
+            margin: 32px
+        }
```

Comment:
Instead of having to do these styles in 3 different files, I recommend we use the `overrides` feature in `theme.js`

https://github.com/setlife-network/trinary/blob/develop/src/styles/theme.js#L55

@sofiaromorales @Ovifer13 thoughts?

Reply:
I was trying to do that in a component with all the components that needed this. Still, I was able to do that in `AllocationAddForm.js` that shares the styles of `AllocationClientSpecifics` and `AllocationAddSpecifics`.

Let me try doing it on `theme.js` and see if it works.